landoapi/models/landing.py

```diff
@@ -0,0 +1,115 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+from landoapi.models.storage import db
+from landoapi.phabricator_client import PhabricatorClient
+from landoapi.transplant_client import TransplantClient
+
+TRANSPLANT_JOB_STARTED = 'started'
+TRANSPLANT_JOB_FINISHED = 'finished'
+
+
+def _get_revision(revision_id, api_key=None):
+    """ Gets revision from Phabricator.
+
+    Returns None or revision.
+    """
+    phab = PhabricatorClient(api_key)
+    revision = phab.get_revision(id=revision_id)
+    if not revision:
+        return None
+
+    raw_repo = phab.get_repo(revision['repositoryPHID'])
+    return {
+        'id': int(revision['id']),
+        'phid': revision['phid'],
+        'repo_url': raw_repo['uri'],
+        'title': revision['title'],
+        'url': revision['uri'],
+        'date_created': int(revision['dateCreated']),
+        'date_modified': int(revision['dateModified']),
+        'status': int(revision['status']),
+        'status_name': revision['statusName'],
+        'summary': revision['summary'],
+        'test_plan': revision['testPlan'],
+    }
+
+
+class Landing(db.Model):
```

Comment:
We should store the ldap_username, date_started, and date_landed. We can make another card for it so that we can land this one quicker.